web/locales/en.json

```diff
@@ -65,7 +65,9 @@
         "lastupdate": "Updated",
         "oops": "Oops! We are having trouble reaching the server. We will try again in a few seconds.",
         "newversion": "A new version is available! Press <a onClick=\"location.reload(true);\">here</a> to reload.",
-        "retrynow": "Retry now"
+        "retrynow": "Retry now",
+        "database-ad": "Looking for historical data? <a href=\"https://data.electricitymap.org\">Check out our Database!</a>",
```

Comment:
Can you add `&utm_source=electricitymap.org&utm_medium=referral` to each link? That's what we've been using so far in the ejs to properly mark the referrals.
Also, should we have `target=_blank`? What do you think?

Reply:
Ah yes, I had completely forgotten our convo about the referral codes, my bad. :) 

Opening in new tab/window is a great idea - I'll add it.